contrib/manyclangs-run

```diff
@@ -18,6 +18,53 @@ print_help() {
     echo 'Otherwise, the exit code is the code returned by COMMAND.'
 }
 
+get_file_arch() {
+    readelf -h "$1" | grep Machine | awk '{$1=""; print $0}' | tr -d ' '
+}
+
+has_docker_image() {
+    if ! docker image inspect "$1" &>/dev/null; then
+        cat
```

Comment:
`cat`? What are the intended semantics of this function? My expectation from the name is that it would be only a test of existence, but this function reads from stdin. Please add an explanatory comment, or tweak, as needed.

Reply:
This is a typo.